http-server/src/main/java/com/facebook/airlift/http/server/HttpServerConfig.java

```diff
@@ -112,6 +112,8 @@
     private int adminPort;
     private int adminMinThreads = 2;
     private int adminMaxThreads = 200;
+    private int schedulerThreads = 1;
```

Comment:
Let's call these two properties the same way they are called in the `HttpClientConfig`, 

```
private int timeoutThreads = 1;
private int timeoutConcurrency = 1;
```